Makefile

```diff
@@ -1,21 +1,30 @@
 # Version tag for all images
 TEST_INFRA_VERSION ?= "latest"
+
 # Version of the gRPC driver
 DRIVER_VERSION ?= "master"
+
 # Prefix for all images used as clone and ready containers, enabling use with registries
 # other than DockerHub
 INIT_IMAGE_PREFIX ?= ""
+
 # Prefix for all images used as build containers, enabling use with registries
 # other than DockerHub
 BUILD_IMAGE_PREFIX ?= ""
+
 # Prefix for all images used as runtime containers, enabling use with registries
 # other than DockerHub
 IMAGE_PREFIX ?= ""
+
 # Image URL to use all building/pushing image targets
 CONTROLLER_IMG ?= ${IMAGE_PREFIX}controller:${TEST_INFRA_VERSION}
+
 # Image URL to use all building/pushing image targets
 CLEAN_IMG ?= ${IMAGE_PREFIX}cleanup:${TEST_INFRA_VERSION}
-#${IMAGE_PREFIX}cleanup_agent:${TEST_INFRA_VERSION}
+
+# Specifies the pool where the controller and cleanup agent must run
```

Comment:
Perhaps "Node pool for system components (controller and cleanup agent)", or "Node pool for system components, i.e. controller and cleanup agent"